### DISCUSSION TITLE
如何获取收到消息的图片类型

### DISCUSSION THREAD

**Discussion by mintianzan** [nonebot/discussions#415] [问答]
直接使用gocq的get_image这个api的话，在其他情况下都能正常检测，在下面情况会出问题：
原图为gif，用户将其添加为表情
这个情况下gocq会认为获取到的图片为jpg类型
那么有没有什么更好的方式可以获取图片数据，特别是图片后缀名，（并且最好能将图片保存到指定位置）呢

**Comment by yanyongyu** [comment]
这个问题请去gocq提问，与nb无关
